Utils/azuretools-core/src/com/microsoft/azuretools/ui/embeddedbrowser/JxBrowserUtil.java

```diff
@@ -22,42 +22,303 @@
 
 package com.microsoft.azuretools.ui.embeddedbrowser;
 
+import com.microsoft.azure.storage.blob.CloudBlobClient;
+import com.microsoft.azure.storage.blob.CloudBlobContainer;
+import com.microsoft.azure.storage.blob.CloudBlockBlob;
+import com.microsoft.azuretools.azurecommons.helpers.NotNull;
+import org.apache.commons.codec.digest.DigestUtils;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.FilenameUtils;
+import org.apache.commons.io.IOUtils;
+
 import javax.swing.*;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
 import java.lang.reflect.Constructor;
 import java.lang.reflect.Method;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.Map;
-
+import java.net.URI;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.*;
+import java.util.concurrent.*;
+import java.util.logging.Logger;
+/**
+ * Use JxBrowser to load URL for Swing, AWT or JavaFX UIs
+ */
 public class JxBrowserUtil {
+    public static final Logger log = Logger.getLogger(JxBrowserUtil.class.getName());
```

Comment:
There is a `com.microsoft.azure.hdinsight.common.logger.ILogger` can be leveraged. And suggest `slf4j` rather than Java `logger`

Reply:
Looked, it seems like other classes in this module is using this logger. So prefer to keep the alignment.

---
In reply to: [156555518](https://github.com/Microsoft/azure-tools-for-java/pull/1296#discussion_r156555518) [](ancestors = 156555518)